apis/flowcollector/v1beta1/flowcollector_types.go

```diff
@@ -239,6 +239,9 @@ type FlowCollectorEBPF struct {
 	// - `FlowRTT` [unsupported (*)]: enable flow latency (RTT) calculations in the eBPF agent during TCP handshakes. This feature better works with `sampling` set to 1.<br>
 	// +optional
 	Features []AgentFeature `json:"features,omitempty"`
+	// Metrics server endpoint configuration for Prometheus scraper
+	// +optional
+	Server MetricsServerConfig `json:"server,omitempty"`
```

Comment:
could it be called `metricsServer`? Either that, or have an intermediate `metrics` struct (it might be useful if we add more metrics settings later)

Reply:
done